### PR TITLE
Update to projects v6

### PR DESCRIPTION
**Pull request by MitchMcQuinn**

### CODE REVIEW

**Review by pmoncada — COMMENTED** [review]
How were these changes added? Did you go line by line? Or are they just copied in?

Just want to make sure how deeply I should check every line v.s. if you just copied it in then it's probably all accurate.

**Comment by MitchMcQuinn** [comment]
> How were these changes added? Did you go line by line? Or are they just copied in?
> 
> Just want to make sure how deeply I should check every line v.s. if you just copied it in then it's probably all accurate.

The google doc with the changes wasn't in markdown format, so I had to go line by line to maintain the correct formatting, links, images, etc...

File: MoonDAO/docs/Projects/Project System.md [hunk 1]
```diff
@@ -30,53 +30,47 @@ MoonDAO Projects are composed of Rocketeers that lead a team of MoonDAO members
...
-
-2. If the proposal requires up-front funding from MoonDAO it must go through DAO Governance as outlined in the Constitution. Multisig should be created before snapshot voting.
+**Step 2: Proposal Submission.** A [Project Proposal](https://moondao.com/propose) ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be posted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go forward for a Senate vote. See our Proposal Process for more information on how proposals are passed.
 
 The Proposal must follow the template provided hereunder, if the proposal does not follow its relevant template, then it will not be considered valid and will not go forward for a Senate vote.
```
Comment (ryand2d) [inline MoonDAO/docs/Projects/Project System.md]: this was eliminated

File: MoonDAO/docs/Projects/Project System.md [hunk 2]
```diff
@@ -30,53 +30,47 @@ MoonDAO Projects are composed of Rocketeers that lead a team of MoonDAO members
...
 
 **2. Attend weekly Senate meetings.** Rocketeers must attend the weekly townhalls and are required to vote on Senate decisions as outlined in the DAO Constitution. They are required to help review incoming project proposals and encouraged to share their thoughts on matters of DAO governance. Failure to attend meetings without notice or not vote in a timely manner will result in penalties to retroactive compensation. **For each missed Senate vote, defined as no vote within 24 hours of the proposal being active, the team will be deducted 5% of their retroactive rewards.** If a Senate member misses more than three votes, the project may be shut down. Special exceptions can be made for members on different time zones where attendance is infeasible.
```
Comment (ryand2d) [inline MoonDAO/docs/Projects/Project System.md]: Should be updated to Leads

File: MoonDAO/docs/Projects/Project System.md [hunk 3]
```diff
@@ -30,53 +30,47 @@ MoonDAO Projects are composed of Rocketeers that lead a team of MoonDAO members
...
 
-Other members of the project with assigned tasks become **Moonsettlers**. These members must:
+Other members of the project with assigned tasks become **Contributors**. These members must:
 
-**1. Provide weekly written updates on progress.** Post updates on work in the “progress” channel under Contribute. **If a moonsettler doesn’t provide a weekly update to the DAO, then 5% of their rewards will be deducted per missed update.**  
+**1. Provide weekly written updates on progress.** Post updates on work in the “progress” channel under Contribute. **If a Contributor doesn’t provide a weekly update to the DAO, then 5% of their rewards will be deducted per missed update.**
```
Comment (ryand2d) [inline MoonDAO/docs/Projects/Project System.md]: 5% of the project leads

File: MoonDAO/docs/Projects/Project System.md [hunk 4]
```diff
@@ -30,53 +30,47 @@ MoonDAO Projects are composed of Rocketeers that lead a team of MoonDAO members
...
-**1. Provide weekly written updates on progress.** Post updates on work in the “progress” channel under Contribute. **If a moonsettler doesn’t provide a weekly update to the DAO, then 5% of their rewards will be deducted per missed update.**  
+**1. Provide weekly written updates on progress.** Post updates on work in the “progress” channel under Contribute. **If a Contributor doesn’t provide a weekly update to the DAO, then 5% of their rewards will be deducted per missed update.**  
 
-## Project Completion
+## Completion
 
-The Project Rocketeer must make sure the following are complete:  
+The Project Lead must make sure the following are complete:  
 
 **1. Complete a [Final Report](https://docs.google.com/document/d/1aa6rMBh1YwYT0IwhlcM8PStAeI0qeO-WN7C6UrLhq2c/edit).** Each project member must include their contributions in the final report. Reports must be submitted to the DAO in the “Submit Report” channel on Discord before the end of the quarter to be included in that quarter’s reward cycle. These reports will be attached on Snapshot for final voting by the DAO.
```
Comment (ryand2d) [inline MoonDAO/docs/Projects/Project System.md]: missing link to submit report channel

File: MoonDAO/docs/Projects/Project System.md [hunk 5]
```diff
@@ -123,65 +118,17 @@ The reward for each Contributor will be calculated as follows:
...
-
-Therefore Bob’s retroactive reward would be 0.15 ETH and 264,895.859375 vMOONEY.
-
-For more example of how the retroactive rewards are calculated, please [click here](MoonDAO%E2%80%99s%20Quarterly%20Rewards.md)  
-
 **Reward Timing and Method**
 
 The ETH will be given as a lump-sum reward within a month of the end of the quarter. 
  
 The MOONEY will be locked for four years and given in the form of delegated vMOONEY that they can unlock after it has vested. For this process MoonDAO uses “delegation safes”. A delegation safe is a multisig wallet that holds the staked voting power that a contributor earns through the quarterly rewards cycle. For more information about delegation safes, such as how to increase your stake or how to delegate your voting power to another address, see the [delegation safe FAQs](https://docs.moondao.com/Archive/reference/Reference-Docs/Delegation-Safe-FAQs).
```
Comment (ryand2d) [inline MoonDAO/docs/Projects/Project System.md]: differs from v6 doc, was truncated

File: MoonDAO/docs/Projects/Project System.md [hunk 6]
```diff
@@ -191,23 +138,23 @@ If Dispute Resolution through MoonDAO’s arbitration procedure is unsuccessful,
...
 ## Miscellaneous FAQs
 
-**Can a project change Rocketeers?** 
+**Can a project change Leads?** 
 
-Yes. If a Rocketeer must step down for whatever reason but the project is not yet completed, an Astronaut may appoint a new Rocketeer to lead the project.
+Yes. If a Lead must step down for whatever reason but the project is not yet completed, an Astronaut may appoint a new Lead to the project.
 
-**Can Rocketeers be external to MoonDAO, or must they be members?**
+**Can Leadss be external to MoonDAO, or must they be members?**
```
Comment (ryand2d) [inline MoonDAO/docs/Projects/Project System.md]: typo here

**Review by ryand2d — APPROVED** [review]

**Review by ryand2d — APPROVED** [review]

**Pull request merged**